apps/web/src/pages/submission/[step].tsx

```diff
@@ -9,34 +10,70 @@ const Submission = () => {
 
   const step = Number(router.query.step);
 
+  // do not scroll to the top of the form on initial load
+  const formRef = useRef<HTMLDivElement>(null);
+  const prevStepRef = useRef<number>(1);
+  useEffect(() => {
+    if (!isNaN(step) && prevStepRef.current !== step && formRef.current) {
+      window.scrollTo({ left: 0, top: formRef.current.offsetTop, behavior: 'smooth' });
```

Comment:
I'm 50/50 on the use of `behavior: 'smooth'`, it works well in some cases but in others it results in an instant scroll to the top of the page and then a smooth scroll downward toward the form which I don't think is the best behaviour